### ISSUE TITLE
/bin/sh: 1: apk: not found when trying to build the sandbox

### ISSUE THREAD

**Issue by LinuxIsCool** [CarperAI/OpenELM#25]
The sandbox doesn't build.

To reproduce, just clone and follow the instructions here: https://github.com/CarperAI/OpenELM/tree/main/elm/sandbox

I expected to build the sandbox with docker.

![image](https://user-images.githubusercontent.com/31582215/204122958-cffdbae4-ee68-448a-bcb6-034a4431df59.png)

I'm running PopOS! Linux

**Comment by LinuxIsCool** [comment]
I made some progress by commenting out that apk line. 

But now I get stuck on pipenv:

```
Step 6/8 : RUN pipenv install --system --deploy                                       
 ---> Running in 896025ba204c                                                         
Your Pipfile.lock (f027e9) is out of date. Expected: (ec7cd2).                        
Usage: pipenv install [OPTIONS] [PACKAGES]...                                         
                                                                                      
ERROR:: Aborting deploy                                                               
The command '/bin/sh -c pipenv install --system --deploy' returned a non-zero code: 2 
```

**Comment by LinuxIsCool** [comment]
OK I got things running. To solve the above, I entered the pipenv shell and installed requirements, but in order to do that, I needed python3.9-distutils.

On deb (popos!):
```
sudo add-apt-repository ppa:deadsnakes/ppa
sudo apt-get update
sudo apt install python3.9-distutils
pipenv shell
pipenv install
./scripts/build.sh
```

**Comment by LinuxIsCool** [comment]
Finally, I had to update docker to use runsc to get the launch running.

```
sudo runsc install
sudo systemctl restart docker
./scripts/launch.sh
```

**Comment by honglu2875** [comment]
@TheExGenesis It is also fixable by just changing `apk` to `apt` (or commenting out as OP mentioned) and update the `Pipfile.lock` for `python:3.9-slim` (we changed the version but forgot to do this).

**Comment by LouisCastricato** [comment]
Is this resolved

**Comment by honglu2875** [comment]
> Is this resolved

Not yet. 

@TheExGenesis do you want to push a quick patch (just in cas you have other considerations for your sandbox)?

**Comment by herbiebradley** [comment]
Fixed by #26
